page-supporters.php

```diff
@@ -69,6 +69,14 @@
                                      'id'=>'14','linkedin'=>'https://www.linkedin.com/company/edgescan-com',
                                      'twitter'=>'https://twitter.com/edgescan'],
 
+                             ['src'=>'../wp-content/themes/cd-theme/static/img/supporters/bullet.png', 'name'=>'Bullet', 'jobDescription'=>'Bullet provide payroll software to ensure this accounting for salaries and employee tax is managed in an accurate and efficient manner.',
+                                     'id'=>'15','linkedin'=>'https://www.linkedin.com/company/bullet-hq',
+                                     'twitter'=>'https://twitter.com/bullethq'],
+
+                             ['src'=>'../wp-content/themes/cd-theme/static/img/supporters/learnupon.png', 'name'=>'Learnupon', 'jobDescription'=>'LearnUpon provide a free  Learning Management system to CoderDojo to make it easy for our users to take and complete our E-Learning training.',
```

Comment:
Typo in `name`, should have an uppercase "U". Also, if we're capitalising "Learning Management", we should probably also include "System". Also, apparently e-learning should not be capitalised at all - https://en.oxforddictionaries.com/definition/e-learning /cc @rosalanghammer